memory/combat_manager.py

```diff
@@ -249,10 +249,10 @@ def read_next_combat_enemy(self: Self) -> None:
                 return
         self.next_combat_enemy = None
 
-    # Helper function for updating itself and ensuring an internal function doesn't run without
-    # the base. This is different than other modules as an attempt to improve performance of the
-    # combat manager module.
     def _should_update(self: Self) -> None:
+        # Helper function for updating itself and ensuring an internal function doesn't run without
```

Comment:
This should be a docstring with """? Or does this syntax also work?

Reply:
I wasn't sure how to rewrite that part to fit the docstring formatting so i at least moved it where the docstring will eventually be.